portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -567,6 +567,46 @@ definition {
 		MenuItem.viewNotPresent(menuItem = "Delete");
 	}
 
+	test CanPublishAfterDeleteOnPublicationAndProduction {
+		task ("Go to site to disable recycle bin and create 2 Pages") {
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
```

Comment:
It's not necessary to disable the recycle since after we enable publications, it will change automatically, can you remove the steps?